archive/tar_test.ts

```diff
@@ -397,7 +397,7 @@ Deno.test("untarLinuxGeneratedTar", async function () {
     const content = expected.content;
     delete expected.content;
 
-    assertEquals(entry, expected);
+    assertEquals({ ...entry }, expected);
```

Comment:
This seems a bit hacky, but given that `TarEntry` is a private type, downcasting it to object in order to keep the assertion behaving as before seems to be an ok trade-off (?).